app/views/templates/invoices/v4/charge.slim

```diff
@@ -456,7 +456,7 @@ html
           - fees.order(:succeeded_at, :created_at).each do |fee|
             tr
               - if fee.charge.percentage? && fee.amount_details.present?
-                - if fee.charge.basic_rate_percentage?
+                - if fee.basic_rate_percentage?
```

Comment:
Can we remove `Charge#basic_rate_percentage?` method maybe?

Reply:
Removed 👍